src/alpha-lab/blink_detection.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Detecting eye blinks using Pupil Lab’s blink detection pipeline"
+description: Perform posthoc and real-time blink detection
+permalink: /alpha-lab/blink-detection/
+tags: [Neon, Realtime Python API]
+---
+
+# Detecting eye blinks using Pupil Lab's blink detection pipeline
+
+<font size = 2>Products used: [Neon](https://pupil-labs.com/products/neon/), [Realtime Python API](https://pupil-labs-realtime-api.readthedocs.io/en/stable/)</font>
+
+<div class="pb-4" style="display:flex;justify-content:left;">
+  <v-img
+    class="rounded"
+    :src="require('../media/alpha-lab/eye_blinks_anim.gif')"
+    max-width=800px
+  >
+  </v-img>
+</div>
+
+### Eye blinks: artifacts and information
+
+The accurate detection of eye blinks serves a vital role in numerous eye tracking applications. Its importance is twofold: firstly, it helps identify time periods affected by blink-related artifacts, and secondly, isolate key physiological, cognitive, and clinical parameters, such as blink rate and blink duration. Blink detection can even be utilized in order to control external devices, for example by detecting a sequence of blinks made in quick succession. In order to show you how to achieve all of this, we have put together this guide.
```

Comment:
Can we try to make the second sentence a bit easier to read? It's a list of lists in the second point.

Reply:
Are we going to be showing the reader how to "achieve all of this" (the previous sentence) or show them how to use blink as signal?

Reply:
in the end, i decided to remove the last sentence. the next paragraph/section again outlines anyway what "the guide shows" and what it "allows you to build", so i think it's better this way.